src/server/scripts/EasternKingdoms/Karazhan/boss_shade_of_aran.cpp

```diff
@@ -77,6 +77,14 @@ enum Groups
     GROUP_DRINKING      = 1
 };
 
+float elementalPositions[4][4] =
```

Comment:
You can use Positions?

Reply:
Position const elementalPos[4]

Reply:
In the DB you mean?

Reply:
Summoncreature uses floats unfortunately

Reply:
Or is it overloaded? I think it's not